### PR TITLE
Add support for NTLM for the microsoft/docker container on Ubuntu

### PR DESCRIPTION
**Pull request by RDIL**
## PR Summary
from #124 

NOTE: package does not seem to be available for ubuntu 14.04

## PR Checklist

- [X] [PR has a meaningful title](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
  - Use the present tense and imperative mood when describing your changes
- [X] [Summarized changes](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [X] [Change is not breaking](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#making-breaking-changes)
- [X] [Make sure all `Dockerfile`, `.sh`, `.ps1` and `.psm1` files have the correct copyright header](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [X] This PR is ready to merge and is not [Work in Progress](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---work-in-progress).
  - If the PR is work in progress, please add the prefix `WIP:` to the beginning of the title and remove the prefix when the PR is ready.
- **Adds a new image or tag**
  - [X] Not Applicable
- **OR**
  - [ ] Update [README.powershellcommunity.md](https://github.com/PowerShell/PowerShell-Docker/blob/master/assets/README.powershellcommunity.md)
  - [ ] Update [vsts-ci.yml](https://github.com/PowerShell/PowerShell-Docker/blob/master/vsts-ci.yml)

### CODE REVIEW

**Comment by RDIL** [comment]
okay I will once I have time

**Comment by TravisEz13** [comment]
I can merge this and you can do it later.  We just need to update the issue.

**Comment by RDIL** [comment]
no I'll do it now, just give me like 20mins
im in the middle of like 10 things right now

**Comment by TravisEz13** [comment]
No worries.  Just didn't know if I didn't have time meant more than the end of the month.

File: release/servicing/ubuntu14.04/meta.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
     "IsLinux" : true,
```
Comment (TravisEz13) [inline release/servicing/ubuntu14.04/meta.json]: All the servicing images can be deleted as the former servicing release is out of support.

Comment (RDIL) [reply release/servicing/ubuntu14.04/meta.json]: Sure, I’ll do it when I get back to my pc

**Comment by TravisEz13** [comment]
One know failure.  I restarted that job.  If it fails again, please rebase from upstream master.

File: release/servicing/ubuntu14.04/meta.json (hunk 1 above)
Comment (RDIL) [reply release/servicing/ubuntu14.04/meta.json]: ALL the servicing images?

**Comment by RDIL** [comment]
I reverted the servicing images, this should be ready for merge.

**Comment by TravisEz13** [comment]
We normally give 24 hours for review.  This isn't urgent.  Ping me when the PR has been open for 24 hours and I'll merge.

**Comment by RDIL** [comment]
Got it.

**Comment by TravisEz13** [comment]
You don't need to worry about it at all.  This is very helpful, keeping this stuff maintained.

**Pull request merged**